Vet fails on go 1.15

When running `make`, go vet fails:

```
pkg/controller/globalroutediscovery/globalroutediscovery_controller.go:296:93: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
pkg/controller/globalroutediscovery/globalroutediscovery_controller.go:301:113: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
pkg/controller/globalroutediscovery/globalroutediscovery_controller.go:306:81: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
pkg/controller/globalroutediscovery/globalroutediscovery_controller.go:311:101: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
make: *** [vet] Error 2
```

This appears be a caused by a [new warning in vet in go 1.15](https://golang.org/doc/go1.15#vet).

The offending code seems to be this:
`log.Error(err, "route's host not matching", "route[0]", route0.Route.Spec.Host, "route["+string(i)+"]", route.Route.Spec.Host)`

More specifically, this code.  `"route["+string(i)+"]"`

--- a/pkg/controller/globalroutediscovery/globalroutediscovery_controller.go
+++ b/pkg/controller/globalroutediscovery/globalroutediscovery_controller.go
@@ -3,6 +3,7 @@ package globalroutediscovery
 import (
 	"context"
 	errs "errors"
+	"fmt"
 	"reflect"
 
 	"strconv"
@@ -291,24 +292,25 @@ func getGlobalDNSRecord(instance *redhatcopv1alpha1.GlobalRouteDiscovery, name s
 	route0 := routeInfos[0]
 	// consistency checks
 	for i, route := range routeInfos {
+		routeNumberString := fmt.Sprintf("route[%d]", i)
 		if route.Route.Spec.Host != route0.Route.Spec.Host {
 			err := errs.New("route's hosts must match")
-			log.Error(err, "route's host not matching", "route[0]", route0.Route.Spec.Host, "route["+string(i)+"]", route.Route.Spec.Host)
+			log.Error(err, "route's host not matching", "route[0]", route0.Route.Spec.Host, routeNumberString, route.Route.Spec.Host)
 			return nil, err
 		}
 		if route.LoadBalancigPolicy != route0.LoadBalancigPolicy {
 			err := errs.New("route's load balancing policy must match")
-			log.Error(err, "route's load balancing policy not matching", "route[0]", route0.LoadBalancigPolicy, "route["+string(i)+"]", route.LoadBalancigPolicy)
+			log.Error(err, "route's load balancing policy not matching", "route[0]", route0.LoadBalancigPolicy, routeNumberString, route.LoadBalancigPolicy)
 			return nil, err
 		}
 		if route.TTL != route0.TTL {
 			err := errs.New("route's TTLs must match")
-			log.Error(err, "route's TTLs not matching", "route[0]", route0.TTL, "route["+string(i)+"]", route.TTL)
+			log.Error(err, "route's TTLs not matching", "route[0]", route0.TTL, routeNumberString, route.TTL)
 			return nil, err
 		}
 		if !reflect.DeepEqual(route.ReadinessCheck, route0.ReadinessCheck) {
 			err := errs.New("route's health checks must match")
-			log.Error(err, "route's health checks not matching", "route[0]", route0.ReadinessCheck, "route["+string(i)+"]", route.ReadinessCheck)
+			log.Error(err, "route's health checks not matching", "route[0]", route0.ReadinessCheck, routeNumberString, route.ReadinessCheck)
 			return nil, err
 		}
 	}

